lib/deb/s3/package.rb

```diff
@@ -55,6 +55,22 @@ def parse_string(s)
       p
     end
 
+    def encoding_flag(control_file)
+      extension = File.extname(control_file)
+
+      # These are all of the valid compression for PPA indices (this
+      # implementation won't support LZMA)
+      # https://wiki.debian.org/DebianRepository/Format#Compression_of_indices
+      case extension
+      when 'gz' then 'z'
+      when 'xz' then 'J'
+      when 'bz2' then 'j'
+      else ''
+      end
+
+      (ext == 'gz') ? 'z' : (ext == 'xz') ? 'J' : (ext == 'bz2') ? 'j' : ''
```

Comment:
This is what the `case` does, right? I think this can be removed?

Reply:
oh whoops! yah - i was playing around and accidentally committed that.